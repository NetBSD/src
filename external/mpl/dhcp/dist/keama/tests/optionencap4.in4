# option definition config

# options
option space foobar;
option space xyz;

option foobar.encap code 1 = encapsulate xyz;
