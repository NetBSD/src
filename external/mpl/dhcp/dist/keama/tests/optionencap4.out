{
  # option definition config
  # options
  "Dhcp4": {
    "option-def": [
      {
        "space": "foobar",
        "name": "encap",
        "code": 1,
        "type": "empty",
        "encapsulate": "xyz"
      }
    ]
  }
}
